api/src/main/resources/swaggercodegen/libraries/okhttp-gson/api.mustache

```diff
@@ -0,0 +1,239 @@
+{{!
+
+This is an AoU-specific extension of the default Swagger-codegen okhttp API implementation template.
+
+The original template was copied from the 2.2.3 release of swagger-codegen; see https://github.com/swagger-api/swagger-codegen/blob/v2.2.3/modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/api.mustache
+for the original file.
+
+AoU-specific additions or replacements are imported via separate files prefixed with "aou",
+e.g. "aouTraceImports" and "aouTracedMethodCall".
+
+}}
+{{>licenseInfo}}
+
+package {{package}};
+
+import {{invokerPackage}}.ApiCallback;
+import {{invokerPackage}}.ApiClient;
+import {{invokerPackage}}.ApiException;
+import {{invokerPackage}}.ApiResponse;
+import {{invokerPackage}}.Configuration;
+import {{invokerPackage}}.Pair;
+import {{invokerPackage}}.ProgressRequestBody;
+import {{invokerPackage}}.ProgressResponseBody;
+{{#performBeanValidation}}
+import {{invokerPackage}}.BeanValidationException;
+{{/performBeanValidation}}
+
+import com.google.gson.reflect.TypeToken;
+
+import java.io.IOException;
+
+{{#useBeanValidation}}
+import javax.validation.constraints.*;
+{{/useBeanValidation}}
+{{#performBeanValidation}}
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
+import javax.validation.executable.ExecutableValidator;
+import java.util.Set;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+{{/performBeanValidation}}
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+{{>aouTraceImports}}
```

Comment:
You _might_ want to have a switch to generate the vanilla template (without the aou calls) for future testing/debugging.  So something like
```
{{#aouTracingEabled}}
{{>aouTraceImports}}
{{/aouTracingEabled}}
``` 
That would let you quickly determine that your changes are scoped to the places you expect. This would become more helpful if there were more than one layer of customizations in play.